src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/DoobieUserRepositoryInterpreter.scala

```diff
@@ -11,24 +10,24 @@ import SQLPagination._
 
 private object UserSQL {
   def insert(user: User): Update0 = sql"""
-    INSERT INTO USERS (USER_NAME, FIRST_NAME, LAST_NAME, EMAIL, PASSWORD, PHONE)
-    VALUES (${user.userName}, ${user.firstName}, ${user.lastName}, ${user.email}, ${user.password}, ${user.phone})
+    INSERT INTO USERS (USER_NAME, FIRST_NAME, LAST_NAME, EMAIL, HASH, PHONE)
```

Comment:
Guessing the flyway migration would also need to be modified

Reply:
This time I added a second migration, so that commits prior to this one would still work.